pkg/session_recording/recordings.jsx

```diff
@@ -450,6 +450,42 @@
     let RecordingList = class extends React.Component {
         constructor(props) {
             super(props);
+            this.sortBy = this.sortBy.bind(this);
+            this.sortList = this.sortList.bind(this);
+            this.drawSortArrow = this.drawSortArrow.bind(this);
+            this.state = {
+                sorting_field: "start",
+                sorting_asc: false,
+            };
+        }
+
+        drawSortArrow() {
+            $('#sort_arrow').remove();
+            let type = this.state.sorting_asc ? "asc" : "desc";
+            let arrow = '<i id="sort_arrow" class="fa fa-sort-' + type + '" aria-hidden="true"></i>';
+            $(this.refs[this.state.sorting_field]).append(arrow);
+        }
+
+        sortBy(event) {
```

Comment:
This is no longer just a `sortBy` function. First of all, it now takes a non-trivial object, a click event as the argument, then extracts the sorting field from it, and it also changes the sorting order. So, I think it should be called something more specific, like `handleColumnClick`.